diamm/helpers/solr_pagination.py

```diff
@@ -148,6 +144,16 @@ def type_list(self):
         return OrderedDict(filtered_facets)
 
     @property
+    def images_count(self):
+        image_count = self.result.facets['facet_fields']['public_images_b']
+        if not image_count:
+            return 0
+        i = image_count.index('true') + 1 if 'true' in image_count else None
```

Comment:
ah. It would be helpful to put that as a comment... I thought you were incrementing the count by 1 and was confused.